internal/node/node.go

```diff
@@ -72,3 +77,40 @@ func (n *node) FindNodeCondition(cond []v1.NodeCondition, conditionType v1.NodeC
 
 	return nil
 }
+
+func (n *node) AddModulesLabelsToNode(modules []types.NamespacedName, node *v1.Node) {
```

Comment:
Who is using this method?

Reply:
my bad, should have deleted it sooner, its fixed now